cypress/support/fragments/users/transferFeeFine.js

```diff
@@ -293,13 +312,13 @@ export default {
       // cy.do(Select({ name: 'header[2].timezone' }).choose('America/New_York'));
 
       cy.do(Select({ name: 'header[3].type' }).choose('Whitespace'));
-      cy.do(TextField({ name: 'header[3].repeat' }).fillIn('1'));
+      cy.do(TextField({ name: 'header[1].repeat' }).fillIn('1'));
```

Comment:
Why is it header[1] instead of header[3] here?

Reply:
@nhanaa Must be an error from when I was resolving the merge conflict. Will fix ASAP!